sci-electronics/minipro/minipro-0.7.recipe

```diff
@@ -48,13 +48,13 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir make $jobArgs
-	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir make library $jobArgs
+	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir/minipro make $jobArgs
+	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir/minipro make library $jobArgs
```

Comment:
DEST_DIR is not needed, except when cross-compiling, I think.

Reply:
Alright thanks, I was not sure about it.